types/meson.build

```diff
@@ -68,3 +68,10 @@ wlr_files += files(
 	'wlr_xdg_decoration_v1.c',
 	'wlr_xdg_output_v1.c',
 )
+if uuid.found() or uuid_create
```

Comment:
Hmm. This means we'll sometimes ship the header files without actually shipping the implementation.

Maybe we can use `exclude_files` in `install_subdir` to prevent this?

Reply:
Bleh, already done